UPGRADE.md

```diff
@@ -0,0 +1,139 @@
+### Kubevirt HCO upgrade procedure
+
+Please note that the instructions were tested w/ OpenShift 4.2.
+Known issues related to the upgrade:
+
+[https://docs.google.com/document/d/1BM8NudsavT4Yh0Mvqv0D-vpqufYhWKAPradphbBB5KQ/edit?usp=sharing](https://docs.google.com/document/d/1BM8NudsavT4Yh0Mvqv0D-vpqufYhWKAPradphbBB5KQ/edit?usp=sharing "https://docs.google.com/document/d/1BM8NudsavT4Yh0Mvqv0D-vpqufYhWKAPradphbBB5KQ/edit?usp=sharing")
```

Comment:
Can you list them out?  This is an internal doc.

Reply:
I wonder if we may still want to share these known issues (current and future ones).
Removing the reference for now.